openapi_python_client/openapi_parser/openapi.py

```diff
@@ -154,24 +154,32 @@ class Schema:
     relative_imports: Set[str]
 
     @staticmethod
-    def from_dict(d: Dict[str, Any], /) -> Schema:
-        """ A single Schema from its dict representation """
+    def from_dict(d: Dict[str, Any], /, name: str = None) -> Schema:
```

Comment:
Doesn't this have to be required (no `= None`)?  If the "title" property is missing the need a `name` to fall back on, otherwise we're trying to create a `Reference` from `None` which would cause an exception.

Reply:
```suggestion
    def from_dict(d: Dict[str, Any], /, name: str) -> Schema:
```